docs/release-notes/release-0-7-2.md

```diff
@@ -0,0 +1,136 @@
+# v0.7.2
+
+Release date: April 08, 2024
+
```

Comment:
@waynexia Maybe manually add some highlights.

IIRC, this version is mainly to resolve the performance regression on compaction scenarios?